docs/docs/cmd/spp/model/model-list.mdx

```diff
@@ -0,0 +1,278 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# m365 spp model list
+
+Retrieve the list of SharePoint Premium unstructured document processing models on the content center site
+
+## Usage
+
+```sh
+m365 spp model list [options]
+```
+
+## Options
+
+```md definition-list
+`-u, --siteUrl <siteUrl>`
+: The URL of the content center site.
+```
+
+<Global />
+
+## Remarks
+
+:::note
+You can only create one Trial Container per tenant, a single application registration can only contain one Container and a tenant can contain a maximum of five Containers in total.
+:::
+
+## Examples
+
+Retrieve the list of SharePoint Premium unstructured document processing models on the content center site.
+
+```sh
+m365 spp model list --siteUrl https://contoso.sharepoint.com/sites/ContentCenter
+```
+## Response
+
+<Tabs>
+  <TabItem value="JSON">
+
+  ```json
+  [
+    {
+      "AIBuilderHybridModelType": "Freeform document processing",
```

Comment:
When showing results, we insist to display only 1 result object so we don't clutter the page too much. Currently we're showing 3 object results. This applies to all response types md, text, csv, ...

Reply:
Done